modules/impersonation-system-admin-user.adoc

```diff
@@ -16,3 +16,16 @@ cluster administrator permissions.
 ----
 $ oc create clusterrolebinding <any_valid_name> --clusterrole=sudoer --user=<username>
 ----
+
+When a `system:admin` user is granted cluster administration permissions via a group, you must include the
+`*--as=<user> --as-group=<group1> --as-group=<group2>*` parameters in the command to impersonate the associated group(s).
```

Comment:
Literal values just use backticks, so the asterisks around the parameters should be removed.

Reply:
Don't use (s). It's fine to just say groups.